src/provisioner/provisioner.py

```diff
@@ -49,7 +49,7 @@ def config_apply(solution_conf_url: str, cortx_conf_url: str = None):
         if Log.logger is None:
             CortxProvisionerLog.initialize(const.SERVICE_NAME, const.TMP_LOG_PATH)
         Log.info('Applying config %s' % solution_conf_url)
-        Conf.load(CortxProvisioner._solution_index, solution_conf_url)
+        Conf.load(CortxProvisioner._solution_index, solution_conf_url, fail_reload=False)
```

Comment:
fail_reload option needs to be derived from the input parameter force_config. 

If force_config is True, we reload the config, i.e. fail_reload=False
if force_config is False, we ignore failure on reload, skip_reload=True

cs_option = {"fail_reload":False} if force_config else {"skip_reload":True}
Conf.load(CortxProvisioner._solution_index, solution_conf_url, cs_option)

Reply:
added force_config option